docker/rustc_info.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
This file is intentionally non-executable and named differently than `rustc.sh` just to avoid any conflicts or confusion with `rustc` itself.